cirq/testing/random_circuit_test.py

```diff
@@ -67,3 +68,31 @@ def test_random_circuit(n_qubits: Union[int, Sequence[cirq.Qid]],
     assert set(cast(cirq.GateOperation, op).gate
                for op in circuit.all_operations()
                ).issubset(gate_domain)
+
+
+@pytest.mark.parametrize('seed', [randint(0, 2**32) for _ in range(10)])
+def test_random_circuit_reproducible_with_integer_seed(seed):
+    circuit1 = random_circuit(qubits=20,
+                              n_moments=20,
+                              op_density=0.7,
+                              random_state=seed)
+    circuit2 = random_circuit(qubits=20,
+                              n_moments=20,
+                              op_density=0.7,
+                              random_state=seed)
+    assert circuit1 == circuit2
+
+
+@pytest.mark.parametrize('seed', [randint(0, 2**32) for _ in range(10)])
+def test_random_circuit_reproducible_with_random_state(seed):
```

Comment:
Given that the parameters of these tests are the same, they should probably just be combined. Something like
```for s in (seed, np.random.RandomState(seed)):```

Reply:
I've combined the tests.